comparisons/navigation/lightbox/a11y.txt

```diff
@@ -0,0 +1 @@
+Usage: Not keyboard navigable.
```

Comment:
I'd lump the lightbox in with the modal. Probably not safe to use as there's a lot of keyboard support that needs to be added